hpke/util.go

```diff
@@ -72,22 +73,38 @@ func (st state) verifyPSKInputs(psk, pskID []byte) error {
 func (suite Suite) String() string {
 	return fmt.Sprintf(
 		"kem_id: %v kdf_id: %v aead_id: %v",
-		suite.KemID, suite.KdfID, suite.AeadID,
+		suite.kemID, suite.kdfID, suite.aeadID,
 	)
 }
 
 func (suite Suite) getSuiteID() (id [10]byte) {
 	id[0], id[1], id[2], id[3] = 'H', 'P', 'K', 'E'
-	binary.BigEndian.PutUint16(id[4:6], uint16(suite.KemID))
-	binary.BigEndian.PutUint16(id[6:8], uint16(suite.KdfID))
-	binary.BigEndian.PutUint16(id[8:10], uint16(suite.AeadID))
+	binary.BigEndian.PutUint16(id[4:6], uint16(suite.kemID))
+	binary.BigEndian.PutUint16(id[6:8], uint16(suite.kdfID))
+	binary.BigEndian.PutUint16(id[8:10], uint16(suite.aeadID))
 	return
 }
 
+func (suite Suite) KDFExtract(secret, salt []byte) (pseudorandomKey []byte) {
```

Comment:
I'm curious: why is this a function of the `Suite`, and not a function of the inner `KDF`? [We already expose KDF-specific functions](https://github.com/armfazh/circl/pull/8/files#diff-8b8e4aa268993805b8226304cb9bd0b7c54f4f952d32148952d7ff43af9bb513R115) as part of this change.

Reply:
Good catch.

Reply:
Updated now